cluster/addons/dashboard/dashboard-controller.yaml

```diff
@@ -1,27 +1,29 @@
+{% set version = "0.1.0-alpha1" -%}
```

Comment:
`dashboard` hasn't been released yet. @bryk knows what value the first beta release will have.
I think a `canary` image is pushed, but haven't checked it